provider/marathon/marathon.go

```diff
@@ -32,6 +33,10 @@ const (
 
 var _ provider.Provider = (*Provider)(nil)
 
+// Regexp used to extract the name of the service and the name of the property for this service
+// All properties are under the format traefik.<servicename>.frontent.*= except the port/portIndex/weight/protocol directly after traefik.<servicename>.
+var servicesPropertiesRegexp = regexp.MustCompile(`^traefik\.(?P<service_name>.*?)\.(?P<property_name>port|portIndex|weight|protocol|frontend\.(.*))$`)
```

Comment:
Yep, good point.  TBH, I just copied the regex from https://github.com/containous/traefik/pull/1257/files#diff-0e8a9f9593585fb3fa7ef0396fd5bbecR319, so should probably change it there as well, right?